networkx/algorithms/link_analysis/hits_alg.py

```diff
@@ -82,34 +82,34 @@ def hits(G,max_iter=100,tol=1.0e-8,nstart=None,normalized=True):
         s=1.0/sum(h.values())
         for k in h:
             h[k]*=s
-    i = 1
-    while i <= max_iter:  # power iteration: make up to max_iter iterations
-        hlast = h
-        h = dict.fromkeys(hlast.keys(), 0)
-        a = dict.fromkeys(hlast.keys(), 0)
```

Comment:
These changes are nice to have. We don't go out and change every file, but whenever we do edit a file, PEP8 changes are okay. So if you want to add these back in, please do (separate commit though). Not necessary though.